src/components/InMoreLanguageSwitch.vue

```diff
@@ -0,0 +1,55 @@
+<template>
+	<div class="wikibase-termbox-subsection-switch">
+		<div class="wikibase-termbox-subsection-switch__switch">
+			<a :href="link" @click="clientShowAllLanguages( $event )">{{ message( MESSAGE_KEYS.ALL_LANG ) }}</a>
```

Comment:
```suggestion
			<a :href="link" @click="clientShowAllLanguages">{{ message( MESSAGE_KEYS.ALL_LANG ) }}</a>
```

The event is passed automagically.

Reply:
see below